elements/bulbs-ellipsize/bulbs-ellipsize.js

```diff
@@ -0,0 +1,22 @@
+import { BulbsHTMLElement, registerElement } from 'bulbs-elements/register';
+import invariant from 'invariant';
+import truncate from 'truncate.js'; // eslint-disable-line no-unused-vars
+
+class BulbsEllipsize extends BulbsHTMLElement {
+  attachedCallback () {
+    invariant(this.getAttribute('line-count'),
+      'BulbsEllipsize: Expects a line-count attribute');
+    if (!this.textContent) { return; }
+
+    this.truncate();
+  }
+
+  truncate () {
+    let lineCount = parseInt(this.getAttribute('line-count'), 10);
+
+    $(this).truncate({ lines: lineCount, ellipsis: '...' });
```

Comment:
The rational is that jQuery is outdated in a code-base like bulbs-elements. There's simply no need to depend on it. I'm not sure what the internals look like for truncate.js but I would bet that it doesn't extend jQuery if it doesn't find it. In our other js code, we're not ready to let go of the almighty `$` but bulbs-elements is stuffed with new hotness. jQuery, in contrast, is old and busted.